pkgs/development/compilers/llvm/common/llvm/default.nix

```diff
@@ -399,6 +402,8 @@ stdenv.mkDerivation (rec {
         nativeInstallFlags
       ])
     )
+  ] ++ optionals enableDebuginfod [
```

Comment:
This should be optional because it's only 1 element.

Reply:
I modeled this according to how it's done in the rest of the file. Should I just break the convention?